spec/node/asciidoctor.spec.js

```diff
@@ -9,11 +9,20 @@ var config = {
   }
 };
 var asciidoctor = require('../../build/asciidoctor.js')(config);
+function asciidoctorVersionGreaterThan (version) {
```

Comment:
In other words, these rules are strict: http://ruby-doc.org/stdlib-2.4.1/libdoc/rubygems/rdoc/Gem/Version.html